src/orientation.rs

```diff
@@ -104,10 +104,10 @@ impl SensorAccelerometer {
 
         // Compute angle to landscape/portrait mode.
         //
-        // This will return the angle between the orientation axis and the position. So
-        // for example when the portrait_angle is zero it means that device is
-        // vertical, however to determine whether it's upside down or vise-versa
-        // we must look at the angle to landscape.
+        // This will return the angle between the orientation axis and the
+        // position. So for example when the portrait_angle is zero it means
+        // that device is vertical, however to determine whether it's upside
```

Comment:
```suggestion
        // that the device is vertical, however to determine whether it's upside
```

Really tiny last thing, but I think without the `the` this just doesn't read nicely.